src/hw-2/index.ts

```diff
@@ -0,0 +1,15 @@
+import express from 'express';
+import * as core from 'express-serve-static-core';
+
+import userRouter from './routers/routes';
+
+const app: core.Express  = express();
+const PORT = 3000;
```

Comment:
I like when app has a way to configure such things like port
`const PORT = process.env.PORT || 3000`